onnxruntime/python/tools/transformers/models/llama/benchmark.py

```diff
@@ -11,7 +11,7 @@
 import onnx
 import psutil
 import torch
-from benchmark_helper import setup_logger
+from onnxruntime.transformers.benchmark_helper import setup_logger
```

Comment:
When using the README instructions and running the scripts as `python -m models.llama.<file> ...`, we can omit this change and leave the import as `from benchmark_helper import setup_logger`

Reply:
This pattern will work for both python -m models.llama.<file> and python benchmark.py ...